examples/credential_issuer_metadata_jwt_vc_json.json

```
@@ -4,6 +4,14 @@
     "credential_endpoint": "https://credential-issuer.example.com",
     "batch_credential_endpoint": "https://credential-issuer.example.com/batch_credential",
     "deferred_credential_endpoint": "https://credential-issuer.example.com/deferred_credential",
+    "credential_response_encryption": {
+        "alg_values_supported" : [
+            "ECDH-ES"
+        ],
+        "enc_values_supported" : [
+            "A128GCM"
+        ]
```

`encryption_required` is marked as 'required' in the spec so should be present here

```suggestion
        ],
        "encryption_required": false
```

---

thanks Joseph